klippy/extras/resonance_tester.py

```diff
@@ -299,18 +334,22 @@ def cmd_MEASURE_AXES_NOISE(self, gcmd):
     def is_valid_name_suffix(self, name_suffix):
         return name_suffix.replace('-', '').replace('_', '').isalnum()
 
-    def get_filename(self, base, name_suffix, axis=None, point=None):
+    def get_filename(self, base, name_suffix, axis=None,
+                     point=None, chip_name=None):
         name = base
         if axis:
             name += '_' + axis.get_name()
+        if chip_name:
+            name += '_' + chip_name
```

Comment:
chip_name may contain spaces (e.g. be "adxl345 toolhead1"). Ideally, you'd want to replace them with something, e.g. call replace(" ", "_")